src/ColouringBook.js

```diff
@@ -207,6 +329,25 @@ export class ColouringBook extends LitElement {
 
 		this.colour = this.paletteColours[0]
 		this.setCursor()
+
+		this.collapseMenuMobile === true ? this.showCollapseMenuMobile() : '';
+
+		this.toolsRightOffset >= 0 ? this.offsetToolsToRight() : '';
+	}
+
+	async _getHistory(image, index) {
+		let x = await window.localStorage.getItem('davie:' + this.identity + this.selectedImage);
+	   x ? this.paths = JSON.parse(x) : this.paths = [];
+		   let canvasThumbCtx=this._getCTX(`can-${index}`);
+		   let imageThumb=this.shadowRoot.getElementById(`img-${index}`)
+		   let ratio = imageThumb.width / this.img.width
+		   
+		   await this.refresh(canvasThumbCtx, imageThumb,ratio )
+	   
+   	}
+   
+	_getCTX(image) {
+	   return this.shadowRoot.getElementById(image).getContext('2d')
```

Comment:
You have repeated an existing two functions

Reply:
Hi @davie-robertson , its fixed and pushed now.